clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -87,6 +89,13 @@
  * Do not add exceptions that occur only on the client or only on the server here.
  */
 public enum Errors {
+    UNKNOWN_RETRIABLE(-2, "The client received an unexpected error code when processing the response",
```

Comment:
Actually I find it better to name the new error as UnknownErrorCode. I will use it in the updated patch. Do you think this name is better?